### PR TITLE
Set to use Net::Netrc for credentials

### PR DESCRIPTION
**Pull request by jeanconn**
Set to use Net::Netrc for credentials for get_iFOT_events and get_web_content occweb queries.

Also remove a broken DATA install in the Makefile and comment out installation of DATA from the Makefile.

Functional testing:

Copied appropriate credentials to kadi user .netrc and ran get_iFOT_events and get_web_content without credential errors.

### CODE REVIEW

**Comment by jeanconn** [comment]
It could be smarter (like pass the occweb string to Net::Netrc lookup), but we're not looking to add smart options on the Perl side.

File: Makefile [hunk 1]
```diff
@@ -16,7 +16,7 @@ SHARE = Event.pm Snap.pm parse_cm_file.pl arc_time_machine.pl \
 	timeline.js timeline.css vert_line.gif \
 	task_schedule.cfg
 
-DATA =	ACE_hourly_avg.npy
+DATA =
```
Comment (taldcroft) [inline Makefile:19]: What's this about? Seems unrelated.

File: get_web_content.pl [hunk 2]
```diff
@@ -37,7 +38,14 @@
...
     # Get username and password from authorization file(s) (which might be a glob)
     if (defined $web_opt{auth_file}) {
-        ($web_opt{user}, $web_opt{passwd}) = Ska::Web::get_user_passwd($web_opt{auth_file})
+	if ($web_opt{auth_file} eq 'occweb'){
+	    my $netrc = Net::Netrc->lookup('occweb');
+	    $web_opt{user} = $netrc->login;
+	    $web_opt{passwd} = $netrc->password;
+	}
+	else{
+	    ($web_opt{user}, $web_opt{passwd}) = Ska::Web::get_user_passwd($web_opt{auth_file})
```
Comment (taldcroft) [inline get_web_content.pl:47]: Why is this still supported? It looks like it is orphaned given the update to `web_content.cfg`, and this flexibility is not in `get_iFOT_events`. We could just get rid of the `auth_file` option entirely?

File: Makefile (hunk 1 above)
Comment (jeanconn) [reply Makefile:19]: So in general I'm left with "do I use my commit messages to store history or just copy them all into the PR text"?  Here, the Makefile doesn't work in a clean repo because the npy is missing, so I stuck that in ee23dbc.  If we want it in a separate PR that's fine.  If we want to update the Makefile so that the process to build a new npy file is run before trying to install it that's fine.

File: get_web_content.pl (hunk 2 above)
Comment (jeanconn) [reply get_web_content.pl:47]: So, I tested that the I could run get_iFOT_events and get_web_content with the current profile.  Checking that nothing else in here was using those pieces and configurables seemed more steps requiring more verification.  If you want that, that's fine too.

Comment (jeanconn) [reply get_web_content.pl:47]: With regard to removing auth_file completely, I suppose that would be a different exercise... I think you'd need to special case occweb in the calling/fetching code if you no longer had something in the config, otherwise you'd send incorrect occweb credentials when doing every other web query.

Comment (jeanconn) [reply get_web_content.pl:47]: Mostly you are just seeing that I was trying to make changes with the smallest scope and possibility for error.

File: Makefile (hunk 1 above)
Comment (taldcroft) [reply Makefile:19]: These kinds of things go into the PR description. I don't mind unrelated changes in a PR like this when we are just trying to get stuff done, but you need to document the changes and rationale in the PR description. If I saw such a description I would have just said "fine" to myself and moved on, though it raises the question of whether a clean install without that file actually works. I don't remember if that file gets auto-created or if some process will just fail because of a missing file.

File: get_web_content.pl (hunk 2 above)
Comment (jeanconn) [reply get_web_content.pl]: If we just care about the current task schedule use case being tested (what I did) then it is fine to clean this up and remove the support of additional/alternative auth_file configurations.  I just replated the web_content.cfg entry with a new netrc keyword (can be renamed as you like) and removed the conditional in here.

**Review by taldcroft — APPROVED** [review]
